notify/notify.go

```diff
@@ -848,6 +848,7 @@ func (r RetryStage) exec(ctx context.Context, l *slog.Logger, alerts ...*types.A
 		default:
 		}
 
+		i++
```

Comment:
Doesn't this mean that if Line 844 is true, the error message contains `0 attempts`, which is incorrect?